src/exceptions.md

```diff
@@ -63,9 +66,7 @@ pub static EXCEPTIONS: [Vector; 14] = [
     Vector { handler: HardFault },
     Vector { handler: MemManage },
     Vector { handler: BusFault },
-    Vector {
```

Comment:
FWIW, the original code was formatted using (nightly) rustfmt with default settings.